flocker/node/_docker.py

```diff
@@ -564,11 +564,15 @@ def _list():
                 image_data = self._client.inspect_image(image)
                 unit_environment = []
                 container_environment = data[u"Config"][u"Env"]
```

Comment:
It was speculated on `#docker` that `ContainerConfig` is stuff inherited by a container that is started with this image.  That makes sense for at least some of the keys present...  It looks like at least for some data the two sections don't contain exactly the same information, but I've only observed a deviation for `Cmd`.  Perhaps it doesn't really matter for this PR, though.